### ISSUE TITLE
install, is it just me?

### ISSUE THREAD

**Issue by mfdoom2** [westonplatter/fast_arrow#38]
Collecting fast_arrow
  Cache entry deserialization failed, entry ignored
  Using cached https://files.pythonhosted.org/packages/ad/3c/a18ad138b7e5c29cd63b3811baa1573af74810001933785b222b0faa3886/fast_arrow-0.2.4-py3-none-any.whl
Requirement already satisfied: yarl in /usr/local/lib/python3.6/dist-packages (from fast_arrow)
Requirement already satisfied: requests in /usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg (from fast_arrow)
Requirement already satisfied: pathlib2 in /usr/local/lib/python3.6/dist-packages (from fast_arrow)
Requirement already satisfied: click in /usr/local/lib/python3.6/dist-packages/click-6.7-py3.6.egg (from fast_arrow)
Requirement already satisfied: idna>=2.0 in /usr/lib/python3/dist-packages (from yarl->fast_arrow)
Requirement already satisfied: multidict>=4.0 in /usr/local/lib/python3.6/dist-packages (from yarl->fast_arrow)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages/certifi-2018.8.13-py3.6.egg (from requests->fast_arrow)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python3.6/dist-packages/chardet-3.0.4-py3.6.egg (from requests->fast_arrow)
Requirement already satisfied: urllib3<1.23,>=1.21.1 in /usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg (from requests->fast_arrow)
Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from pathlib2->fast_arrow)
Installing collected packages: fast-arrow
Successfully installed fast-arrow-0.2.4
root@ip:/home/ubuntu/FAST# python3.6 fast.py
Fatal Python error: Cannot recover from stack overflow.

Current thread 0x00007fe51c9d4740 (most recent call first):
  File "/usr/lib/python3.6/threading.py", line 226 in __init__
  File "/usr/lib/python3.6/queue.py", line 39 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/connectionpool.py", line 181 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/connectionpool.py", line 769 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 195 in _new_pool
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 261 in connection_from_pool_key
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 240 in connection_from_context
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 227 in connection_from_host
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 279 in connection_from_url
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/adapters.py", line 309 in get_connection
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/adapters.py", line 405 in send
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/sessions.py", line 618 in send
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/sessions.py", line 508 in request
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/api.py", line 58 in request
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/api.py", line 112 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 75 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  ...
Aborted

Current thread 0x00007f6d8a294740 (most recent call first):
  File "/usr/lib/python3.6/threading.py", line 226 in __init__
  File "/usr/lib/python3.6/queue.py", line 39 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/connectionpool.py", line 181 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/connectionpool.py", line 769 in __init__
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 195 in _new_pool
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 261 in connection_from_pool_key
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 240 in connection_from_context
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 227 in connection_from_host
  File "/usr/local/lib/python3.6/dist-packages/urllib3-1.22-py3.6.egg/urllib3/poolmanager.py", line 279 in connection_from_url
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/adapters.py", line 309 in get_connection
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/adapters.py", line 405 in send
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/sessions.py", line 618 in send
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/sessions.py", line 508 in request
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/api.py", line 58 in request
  File "/usr/local/lib/python3.6/dist-packages/requests-2.18.4-py3.6.egg/requests/api.py", line 112 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 75 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 148 in relogin_oauth2
  File "/usr/local/lib/python3.6/dist-packages/fast_arrow/client.py", line 81 in post
  ...

**Comment by westonplatter** [comment]
@mfdoom2 i'm hitting the same issue when I install from pypi; if you git clone the code and install it form source, the samples should still work.

**Comment by westonplatter** [comment]
@mfdoom2 looked into this, and found I didn't include the root SSL certs that python requests checks. Whoops.

I think I fixed the issue via PR #37. I released `0.2.5a2` to Pypi, installed it locally, and things are working for me. Any chance you'd be willing to test it for me before I release a new version? 

I had to pip install the alpha version like this, 

```
pip install https://files.pythonhosted.org/packages/c9/41/441d8d479c0e3a2f4db3713ab9b928ebf9e52d96808dd34f5818f6fc64b9/fast_arrow-0.2.5a2-py3-none-any.whl```
